### PR TITLE
kubesphere-3.1.0-ga-announcement.md

### PR DESCRIPTION
**Pull request by faweizhao26**
Signed-off-by: faweizhao26 <faweizhao@yunify.com>

### CODE REVIEW

File: content/zh/blogs/kubesphere-3.1.0-ga-announcement.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+统一的身份管理和完备的鉴权体系，是多租户系统中实现逻辑隔离不可或缺的能力。KubeSphere 基于 Kubernetes 的角色访问控制（RBAC），提供了细粒度的权限控制能力，在支持 Namespace 层级资源隔离的同时通过 Workspace 进一步对租户进行了定义，多层级的权限管控更加契合企业用户的使用场景。
+
+v3.1.0 中新增了组织架构管理功能，可以通过用户组简化批量授权操作。除此之外还支持了企业空间的资源配额管理，实现对资源用量的管控，进一步满足企业用户的实际需求。
+
+统一认证方面，v3.1.0 中简化了身份提供商（IdentityProvider, IdP）的配置方式，除 LDAP 之外新增了对 CAS（Central Authentication Service）、OIDC（OpenID Connect）、OAuth2 等通用认证协议的支持，并提供了插件化的拓展方式，以便不同账户系统之间的集成。
+
+<iframe width="760" height="380" src="//player.bilibili.com/player.html?aid=802754514&bvid=BV1jy4y1s7i3&cid=331412102&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
+
+<iframe width="760" height="380" src="//player.bilibili.com/player.html?aid=972796031&bvid=BV1qp4y1b7Jz&cid=331391855&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
```
Comment (LinuxSuRen) [inline content/zh/blogs/kubesphere-3.1.0-ga-announcement.md:111]: image

It looks like it does not work well here.

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Thanks for the great news. I left some comments, please have a look at it.

File: content/zh/blogs/kubesphere-3.1.0-ga-announcement.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+在企业运营和管理 Kubernetes 容器平台时，通常需要分析资源消耗，查看集群及其中租户的消费账单，洞察资源使用情况，分析基础设施运营成本。
+
+在 KubeSphere 3.1.0 中，可从多个维度来分析平台资源消耗：
+
+- 从集群维度，可查看每个集群资源消耗，深入到节点中分析运行的工作负载，精准规划每个节点中工作负载的资源使用状况。
+- 从企业空间维度，可查看每个企业空间资源消耗，获取企业空间中项目、应用、工作负载的消费账单，分析多租户环境中各个租户的资源使用是否合理。
+
+另外，除了可以通过界面查看和导出数据，KubeSphere 计量计费平台也提供了所有操作的 API。接下来在后续的版本里，会持续加强并构筑端到端完整的计量计费可运营系统。
+
+![](https://cdn.jsdelivr.net/gh/yangchuansheng/imghosting1@main/img/20210429141959.png)
```
Comment (FeynmanZhou) [inline content/zh/blogs/kubesphere-3.1.0-ga-announcement.md:38]: Add the file name in `![xxx]`

File: content/zh/blogs/kubesphere-3.1.0-ga-announcement.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+<iframe width="760" height="380" src="//player.bilibili.com/player.html?aid=587836535&bvid=BV1zB4y1c7oK&cid=330894124&page=1" scrolling="no" border="0" frameborder="no" framespacing="0" allowfullscreen="true"> </iframe>
+
+
+## 强化微服务治理能力
+
+KubeSphere 基于 [Istio](https://github.com/istio/istio) 提供了金丝雀发布、蓝绿部署、熔断等流量治理功能，同时还支持可视化呈现微服务之间的拓扑关系，并提供细粒度的监控数据。在分布式链路追踪方面，KubeSphere 基于 Jaeger 让用户快速追踪微服务之间的通讯情况，从而更易于了解微服务的请求延迟、性能瓶颈、序列化和并行调用等。
+
+KubeSphere 3.1.0 对微服务治理功能进行了强化，将 Istio 升级到了 1.6.10，支持图形化流量方向检测，图像化方式显示应用流量的流入/流出。同时还支持对 Nginx Ingress Gateway 进行监控，新增 Nginx Ingress Controller 的监控指标。
+
+![](https://cdn.jsdelivr.net/gh/yangchuansheng/imghosting@three/img/20210428221044.png)
```
Comment (FeynmanZhou) [inline content/zh/blogs/kubesphere-3.1.0-ga-announcement.md:59]: Please do not use personal CDN storage. You can store these images in this repository or our QingStor Object Storage.

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**